### PR TITLE
Application for socbay.io

### PR DESCRIPTION
**Pull request by nghianguyen119**
# Grant Application Checklist

- [x] The [application-template.md](https://github.com/crustio/Crust-Grants-Program/blob/main/applications/application_template.md) has been copied, renamed ( "project_name.md") and updated.
- [x] A Ethereum (USDT/CRU) address for the payment of the milestones is provided inside the application.  
- [x] The software of the project will be released under the Apache license version 2.0 as specified in the [terms and conditions](https://github.com/crustio/Crust-Grants-Program/blob/main/crust-grants-terms-and-conditions.md).
- [x] I accept the [terms and conditions](https://github.com/crustio/Crust-Grants-Program/blob/main/crust-grants-terms-and-conditions.md) and submit the PR.
- [x] The total funding amount of the project is below USDT $30k.
- [x] The initial PR contains only one commit (squash if needed before submitting your PR).
- [x] The grant will only be announced once we've successfully [delivered](https://github.com/crustio/Crust-Grant-Milestone-Delivery) the first milestone.

### CODE REVIEW

File: applications/crustlive.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
+# Crust Grant Proposal
+
+* **Project Name:** CrustLive
+* **Website:** http://crust.live
```
[the remark this answers is not in the document]
Comment (nghianguyen119) [reply applications/crustlive.md:4]: I'll find another name for the project.

[the remark this answers is not in the document]
Comment (yyd106) [reply applications/crustlive.md:4]: Thanks, and please also set USDT/USDB to USDT only

[the remark this answers is not in the document]
Comment (nghianguyen119) [reply applications/crustlive.md]: Hi, I changed the project name to socbay.io

[the remark this answers is not in the document]
Comment (zikunfan) [reply applications/crustlive.md]: > Hi, I changed the project name to socbay.io

Thank you, we'll arrange DCF tech guys to review your application.

File: applications/socbayio.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+* **Total Estimated Duration:** 4 months
+* **Full-Time Equivalent (FTE):**  1
+* **Total Costs:** 15000 USD
+
+### Milestone 1: MVP website
+* **Estimated Duration:** 2 months
+* **FTE:**  1
+* **Costs:** 10,000 USD
+
+| Number | Deliverable | Specification |
```
Comment (LowEntropyBody) [inline applications/socbayio.md:82]: There is no Crust-related work in your first milestone which alone takes up 1w USD. We recommand that you integrate the feature of Crust storage order in your platform for the first milestone.

File: applications/socbayio.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+* **Estimated Duration:** 2 months
+* **FTE:**  1
+* **Costs:** 5,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 1. |Private gateway|Create private gateway|
+| 2. |Additional features|Implement additional functions for the website (e.g. chat)| 
+| 3. |Website Security|Strengthen the security to avoid malicious attacks|
+| 4. |Free uploading video|Using donation to create a common budget where users could use to store their video easily with Socbay.io|
```
Comment (LowEntropyBody) [inline applications/socbayio.md:100]: Design the CRU payment system. Make it easier for users to use and highlight the concept of decentralized video sites.

File: applications/socbayio.md (hunk 2 above)
Comment (nghianguyen119) [reply applications/socbayio.md:82]: Agree. I made the trial on this part and it's possible to integrate for the first milestone.

**Comment by yyd106** [comment]
Why this is not checked?

**Review by yyd106 — APPROVED** [review]

**Pull request merged**